src/tasks/WasmAppBuilder/EmitWasmBundleFiles.cs

```diff
@@ -0,0 +1,268 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+
+namespace Microsoft.WebAssembly.Build.Tasks;
+
+public class EmitWasmBundleFiles : Microsoft.Build.Utilities.Task, ICancelableTask
```

Comment:
I set it to `Low`. It will show the progress only for new files or clean build, because this is incremental.
It makes sense to report progress for bundling assemblies which are large and slow.